fhir-parent/pom.xml

```diff
@@ -498,23 +498,47 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-project-info-reports-plugin</artifactId>
-                    <version>2.9</version>
+                    <version>3.0.0</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.1.1</version>
+                    <version>3.0.1</version>
```

Comment:
Sorry for the late comment; can we leave a comment on here to explain why we shouldn't upgrade to 3.1

Reply:
Let me put this in my latest PR